Makefile

```diff
@@ -24,15 +24,11 @@ dist: #bin
 # subnet sets up the require subnet for testing on darwin (osx) - you must run
 # this before running other tests if you are on osx.
 subnet:
-	@sh -c "'$(CURDIR)/scripts/setup_test_subnet'"
+	@sh -c "'$(CURDIR)/scripts/setup_test_subnet.sh'"
 
-# test runs the test suite and vets the code
+# test runs the test suite
 test: subnet generate
-	go test $(TEST) $(TESTARGS) -timeout=30s -parallel=4
-
-# testinteg runs the integration test suite
-testinteg: subnet generate
-	INTEG_TESTS=yes go test $(TEST) $(TESTARGS)
+	go list $(TEST) | xargs -n1 go test $(TESTARGS)
```

Comment:
The `-n1` will pull the packages off one at a time and feed them to a separate `go test` command in serial. Otherwise, I think `go test` will run tests for each package in parallel according to max procs.